makefile

```diff
@@ -0,0 +1,36 @@
+.ONESHELL:
+
+nial: nialbasic
+	rm -rf BuildNial/build/*
+	rm -rf BuildNial/src/*
+	cd BuildNial/pkgblder
+	./nial_basic -defs addfeatures
+	cd ../build
+	cmake ../src
+	make
+	mv nial ../../
+	
+setup:
+	mkdir -p BuildCore/build
+	mkdir -p BuildNial/build
+	mkdir -p BuildNial/src
+
+nialcore: setup
+	cd BuildCore/build && cmake ../src && make
+	mv nialcore ../../
+
+nialbasic: nialcore
+	mv BuildCore/build/nialcore BuildNial/pkgblder/nial_basic
```

Comment:
This line is doing nothing, and probably just errors

Reply:
I downloaded a clean Nial_Development and tried the makefile with the following results

[100%] Linking C executable nialcore
make[3]: Leaving directory '/home/johng/Nial_Projects/Nial_Development/BuildCore/build'
[100%] Built target nialcore
make[2]: Leaving directory '/home/johng/Nial_Projects/Nial_Development/BuildCore/build'
make[1]: Leaving directory '/home/johng/Nial_Projects/Nial_Development/BuildCore/build'
mv BuildCore/build/nialcore BuildNial/pkgblder/nial_basic
cd BuildNial/pkgblder
./nialcore -defs buildfromcore
cd ../build
cmake ../src
make
cp nial ../pkgblder/nial_basic
mv nial ../../nialbasic
mv: cannot stat 'BuildCore/build/nialcore': No such file or directory
/bin/sh: 3: ./nialcore: not found
CMake Error: The source directory "/home/johng/Nial_Projects/Nial_Development/BuildNial/src" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
make[1]: Entering directory '/home/johng/Nial_Projects/Nial_Development/BuildNial/build'
make[1]: *** No targets specified and no makefile found.  Stop.
make[1]: Leaving directory '/home/johng/Nial_Projects/Nial_Development/BuildNial/build'
cp: cannot stat 'nial': No such file or directory
mv: cannot stat 'nial': No such file or directory
make: *** [makefile:23: nialbasic] Error 1